### PR TITLE
feat: update topBar's searchBar's style

### PR DESCRIPTION
**Pull request by MaximeJineMaraval**
## Changes description

Update the style of the searchBar (in the topBar) to fit Figma style

## Context

Align Android widget style with Figma style

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on a phone device/emulator.
- [x] I have tested on a tablet device/emulator.
- [x] I have tested on a large screen device/emulator.
- [x] If it includes design changes, please ask for a review with a core team designer.

## Does this introduce a breaking change?

- No

## Other information

##Before

### CODE REVIEW

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/37f6b776-d38a-48f3-af8f-d172a502a2c6/artifact/ecf6e32488204c26/p/3ac9cdc370698a98896cc329afe8795e

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/b395af5d-dda4-4be0-ba27-58a5eb1da8f5/artifact/1bd73054afd5953a/p/7abee9c4a46fe85cb52cea2ad7c66a22

File: appbars/src/main/res/values/styles-topbars.xml [hunk 1]
```diff
@@ -57,20 +57,24 @@
...
 
     <style name="ThemeOverlay.Vitamin.TopBar.Search" parent="">
-        <item name="colorOnPrimary">?attr/vtmnContentColorTertiary</item>
+        <item name="colorOnPrimary">?attr/vtmnContentColorPrimary</item>
+        <item name="android:textColorHint">?attr/vtmnContentColorTertiary</item>
     </style>
 
     <style name="Widget.Vitamin.SearchView" parent="@style/Widget.AppCompat.SearchView">
         <item name="closeIcon">@drawable/ic_vtmn_close_line</item>
-        <item name="searchIcon">@drawable/ic_vtmn_search_line</item>
```
Comment (GerardPaligot) [inline appbars/src/main/res/values/styles-topbars.xml:70]: Does it mean that  `ic_vtmn_search_line` is no more the drawable used as icon at the left of the top app bar?

File: appbars/src/main/res/values/styles-topbars.xml [hunk 2]
```diff
@@ -57,20 +57,24 @@
...
+        <item name="searchIcon">@null</item>
         <item name="searchHintIcon">@drawable/ic_vtmn_search_line</item>
         <item name="goIcon">@drawable/ic_vtmn_check_line</item>
         <item name="voiceIcon">@drawable/ic_vtmn_mic_line</item>
         <item name="iconifiedByDefault">false</item>
+        <item name="android:queryBackground">@android:color/transparent</item>
+        <item name="queryBackground">@android:color/transparent</item>
+        <!-- This negative padding should be removed when a clean way to remove start space will be found -->
+        <item name="android:paddingLeft">-8dp</item>
```
Comment (GerardPaligot) [inline appbars/src/main/res/values/styles-topbars.xml:78]: Surprised by this line. Vitamin is using the Material padding in the design specification. Does it mean that XML Android styles doesn't follow Material specification? 🤔

**Review by GerardPaligot — CHANGES_REQUESTED** [review]

File: appbars/src/main/res/values/styles-topbars.xml (hunk 1 above)
Comment (MaximeJineMaraval) [reply appbars/src/main/res/values/styles-topbars.xml:70]: The searchView usage in Vitamin is: 
if you click the `search` icon in the topAppBar option, it opens a new screen with the searchView deployed.
In this case (and this is right now the only case), the search icon should not be visible in the searchView as there is already an icon at the left: the back icon. You can see the behavior in the screenshots

File: appbars/src/main/res/values/styles-topbars.xml (hunk 2 above)
Comment (MaximeJineMaraval) [reply appbars/src/main/res/values/styles-topbars.xml:78]: This case is complicated. The standalone searchView and the topAppBar embedded searchView have really different looks.

When the searchView is embedded in the topAppBar, without specifying anything, the topAppBar adds extra space between the topAppBar left icon and the searchView. 
Additionally to that, the searchView includes extra start and end spacing. 
The result of that is a huge space between the back icon and the searchView (bigger than the space between the back icon and the classic title)

This "hack" must be temporary, a way to reduce this spacing to look more like the spacing between the back icon and the title in a regular topAppBar.
**It should be removed when a solution will be found** (find a way to say the topAppBar to not force spacing when we add a searchView?)

**Comment by dktbitrise** [comment]
### You can download the apk here :
**sample-debug.apk :** 
https://app.bitrise.io/app/62ac2962b2dd627a/build/0089d69b-f5e0-49df-974d-0063a2be8e39/artifact/8775acd98931041d/p/c7c80b3df6fb66026abe06649eb0d7e7

**Review by GerardPaligot — APPROVED** [review]

**Pull request merged**